client/src/modules/list/components/Items/ItemsList.jsx

```diff
@@ -36,20 +36,6 @@ class ItemsList extends Component {
       : toggle(isOrdered, id, listId);
   };
 
-  voteForItem = item => () => {
```

Comment:
what happened with voting??

Reply:
this method was moved to ListItem. It was necessary because we invoked this method in VotingBox and invoked action (setVote or clearVote) in itemsList, and there was no way to set pending in ListItem, where is all logic related to pending, aborting promises etc.